### PR TITLE
[native]Prestissimo exception error code conversion fix

### PR DESCRIPTION
**Pull request by xiaoxmeng**
Verified on Meta internal runs

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Review by amitkdutta — DISMISSED** [review]
Thanks @xiaoxmeng for fixing the erorr code.

**Review by gggrace14 — DISMISSED** [review]

File: presto-native-execution/presto_cpp/main/common/Exception.h [hunk 1]
```diff
@@ -47,13 +47,17 @@ class VeloxToPrestoExceptionTranslator {
         kTranslateMap = {
             {velox::error_source::kErrorSourceRuntime,
              {{velox::error_code::kMemCapExceeded,
-               {0x00020000,
-                "GENERIC_INSUFFICIENT_RESOURCES",
+               {0x00020007,
+                "EXCEEDED_LOCAL_MEMORY_LIMIT",
                 protocol::ErrorType::INSUFFICIENT_RESOURCES}},
               {velox::error_code::kMemAborted,
```
Comment (rschlussel) [inline presto-native-execution/presto_cpp/main/common/Exception.h:53]: what kind of error is kMemAborted? Should it also be EXCEEDED_LOCAL_MEMORY_LIMIT?

**Review by rschlussel — APPROVED** [review]
These changes look good. It would be nice to convert some of the other GENERIC_XXX_ERRORs to something more specific too.  e.g. kInvalidArgument sounds like it should be INVALID_FUNCTION_ARGUMENT error code.  There are a few kinds of arithmetic exceptions we have that might be appropriate for kArithemticError (e.g. NUMERIC_VALUE_OUT_OF_RANGE, DIVISION_BY_ZERO, or if needed we can create a new error code)

**Review by spershin — APPROVED** [review]

File: presto-native-execution/presto_cpp/main/common/Exception.h (hunk 1 above)
Comment (xiaoxmeng) [reply presto-native-execution/presto_cpp/main/common/Exception.h:53]: This is specific to Velox and will create a new code.

**Comment by xiaoxmeng** [comment]
> These changes look good. It would be nice to convert some of the other GENERIC_XXX_ERRORs to something more specific too. e.g. kInvalidArgument sounds like it should be INVALID_FUNCTION_ARGUMENT error code. There are a few kinds of arithmetic exceptions we have that might be appropriate for kArithemticError (e.g. NUMERIC_VALUE_OUT_OF_RANGE, DIVISION_BY_ZERO, or if needed we can create a new error code)

Sure, will do.

**Pull request merged**